testing/matrix/linter/rules/roles/placement.go

```diff
@@ -157,6 +159,16 @@ func objectRBACPlacementServiceAccount(m utils.Module, object storage.StoreObjec
 			"Name of ServiceAccount should be equal to %q or %q",
 			serviceAccountName, expectedServiceAccountName,
 		)
+	case strings.HasSuffix(shortPath, "service-account.yaml"):
```

Comment:
Let's not add new files to existing rules. It seems fine to me to stick to the `rbac-for-us/rbac-to-us` rule. More files = more entropy. Why do you need to put service accounts in a separate file?

Reply:
I tough that it was a good idea because this was the first case when Deckhouse generates SA that will be used by end user. But I agree that it creates more entropy. Fixed.